moonshot/integrations/cli/redteam/redteam.py

```diff
@@ -88,3 +103,26 @@ def do_delete_context_strategy(self, args: argparse.Namespace) -> None:
     @cmd2.with_argparser(delete_attack_module_args)
     def do_delete_attack_module(self, args: argparse.Namespace) -> None:
         delete_attack_module(args)
+
+    @cmd2.with_argparser(add_bookmark_prompt_args)
```

Comment:
can we change this to add_bookmark_args from add_bookmark_prompt_args

Reply:
updated